javascript/javascript1/week1/main.js

```diff
@@ -0,0 +1,77 @@
+// Age-ify (A future age calculator)
+const yearOfBirth = 1986;
+let yearFuture = 2027;
+let age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture + ".");
```

Comment:
Clean and concise code 👌
Great attention to detail making the yearOfBirth a const and adding a dot to the end of the log. 💫